Makefile

```diff
@@ -252,6 +260,9 @@ run-compose: images/docker-compose.yml images/version.yml
 	docker-compose -f $< run storage-init sh -c 'rm -rf /run/* /config/* ; cp -Lr /conf/* /config/ ; echo IMGA > /run/eve.id'
 	docker-compose -f $< up
 
+run-proxy:
+	ssh $(SSH_PROXY) -N -i $(SSH_KEY) -p $(SSH_PORT) -o StrictHostKeyChecking=no -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null root@localhost &
```

Comment:
When does the backgrounded ssh exit? When make exits?

Reply:
background ssh exists when the image exits -- that's pretty reliable on my Mac OS but I would appreciate if you could test it on Linux @eriknordmark